libs/components/editor-core/src/editor/clipboard/copy.ts

```diff
@@ -0,0 +1,102 @@
+import { Editor } from '../editor';
+import { SelectInfo } from '../selection';
+import { OFFICE_CLIPBOARD_MIMETYPE } from './types';
+import { Clip } from './clip';
+import ClipboardParse from './clipboard-parse';
+import { getClipDataOfBlocksById } from './utils';
+
+class Copy {
```

Comment:
The class name should use a noun that is easy to understand, maybe `CopyHelper` or `CopyOperator` is a good choice?